### PR TITLE
-added documentation for necessary steps to implement a custom plugin

### PR DESCRIPTION
**Pull request by huibvanstraten**
Current documentation is missing some important steps

### CODE REVIEW

File: extending-valtimo/plugin/custom-plugin-definition.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ Valtimo offers the functionality needed to create and add plugins to Valtimo imp
 
 ### Creating a plugin definition
 
+#### Dependencies
+
+To create a custom plugin in your project, the following dependencies are needed:
+- valtimo:plugin
+- valtimo:documenten-api
```
Comment (marijnritense) [inline extending-valtimo/plugin/custom-plugin-definition.md:13]: Only `valtimo:plugin`  is necessary. The others are not.

Comment (huibvanstraten) [reply extending-valtimo/plugin/custom-plugin-definition.md:13]: @marijnritense actually Theo and I found out we need them all.

Comment (marijnritense) [reply extending-valtimo/plugin/custom-plugin-definition.md:13]: Do you have the exceptions? Assuming what you're saying is true, the current state of plugins is unacceptable...

Comment (marijnritense) [reply extending-valtimo/plugin/custom-plugin-definition.md:13]: Smartdocuments works without these dependencies by the way, and contains the smartdocuments plugin.

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**